seatunnel-formats/seatunnel-format-compatible-debezium-json/src/main/java/org/apache/seatunnel/format/compatible/debezium/json/CompatibleDebeziumJsonSerializationSchema.java

```diff
@@ -30,15 +30,20 @@
 @RequiredArgsConstructor
 public class CompatibleDebeziumJsonSerializationSchema implements SerializationSchema {
 
+    private final boolean isKey;
     private final int index;
 
     public CompatibleDebeziumJsonSerializationSchema(SeaTunnelRowType rowType, boolean isKey) {
+        this.isKey = isKey;
         this.index = rowType.indexOf(isKey ? FIELD_KEY : FIELD_VALUE);
     }
 
     @Override
     public byte[] serialize(SeaTunnelRow row) {
         String field = (String) row.getField(index);
+        if (isKey && field == null) {
+            return null;
+        }
```

Comment:
how about this? Because if we not return null, the NPE will be throwed.
```suggestion
        if (field == null) {
            return null;
        }
```

Reply:
@Hisoka-X  value cannot be null